packages/libs/react-ui/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,100 @@
+import { SystemIcon } from '../Icons';
+
+import {
+  cardTitleClass,
+  colorVariants,
+  containerClass,
+  contentClass,
+  expandClass,
+  footerClass,
+  headerContainerClass,
+  iconContainerClass,
+  iconContainerFullWidthClass,
+  simpleClass,
+  simpleContentClass,
+} from './Notification.css';
+
+import classNames from 'classnames';
+import React, { FC } from 'react';
+
+export interface INotificationProps {
+  icon?: typeof SystemIcon[keyof typeof SystemIcon];
+  title: string;
+  children?: React.ReactNode;
+  displayCloseButton?: boolean;
+  expand?: boolean;
+  color?: 'default' | keyof typeof colorVariants;
+  simple?: boolean;
+}
+
+export const Notification: FC<INotificationProps> = ({
+  icon,
+  title,
+  children,
+  displayCloseButton,
+  color = 'default',
+  simple,
+  expand = false,
+}) => {
+  const isSimple = (simple as boolean) || children === undefined;
+  const Icon = icon || SystemIcon.HelpCircle;
+
+  return (
+    <div
+      className={classNames(
+        [color === 'default' ? containerClass : colorVariants[color!]],
+        {
+          [expandClass]: expand,
+          [simpleClass]: isSimple,
+        },
+      )}
+    >
+      <div className={iconContainerClass}>
+        <Icon size={'md'} />
+      </div>
+
+      <div>
+        {isSimple ? (
+          <p className={simpleContentClass}>{title}</p>
+        ) : (
+          <p className={contentClass}>
+            <h4 className={cardTitleClass}>{title}</h4>
+            <p>{children}</p>
+          </p>
+        )}
+      </div>
+
+      {!isSimple && (
+        <div
+          className={expand ? iconContainerFullWidthClass : iconContainerClass}
+        >
+          <div className={headerContainerClass} />
+          {displayCloseButton && <SystemIcon.Close size={'md'} />}
+        </div>
+      )}
+    </div>
+  );
+};
+
+export interface INotificationFooterProps {
+  children: React.ReactNode;
+}
+
+export const NotificationFooter: FC<INotificationFooterProps> = ({
```

Comment:
As discussed in a meeting with the UI team, we want to use the sub component naming convention. So for example, when using these together, we would reference the components as follows: Notification, Notification.Footer, Notification.Header